dev-rust/cargo_c/cargo_c_bin-0.9.27.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="Cargo applet to build and install C-ABI compatibile dynamic and static libraries"
+DESCRIPTION="It produces and installs a correct pkg-config file, a static \
+library and a dynamic library, and a C header to be used by any C (and \
+C-compatible) software."
+HOMEPAGE="https://github.com/lu-zero/cargo-c"
+COPYRIGHT="2020 Luca Barbato"
+LICENSE="MIT"
+REVISION="1"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	srcGitRev="cdd0b630eedd53ae6d89cfec9ee0cea8fa83b124"
+	SOURCE_URI="https://github.com/Begasus/cargo-c/archive/$srcGitRev.tar.gz"
+	CHECKSUM_SHA256="39da8945db1ee54f051ea69b04d2c0a9c765baffab2c4871427c1d1f96b29e03"
+	SOURCE_DIR="cargo-c-$srcGitRev/32bit"
+elif [ "$targetArchitecture" = x86_64 ]; then
+	srcGitRev="a703bd97a18326396b7805aa75a6444796fe4605"
+	SOURCE_URI="https://github.com/Begasus/cargo-c/archive/$srcGitRev.tar.gz"
+	CHECKSUM_SHA256="0af0cef06db1aa7d46ca6ce60ad89bc1ca0c3705c51a7f80f6f87e0b2b84d8f7"
+	SOURCE_DIR="cargo-c-$srcGitRev/64bit"
+fi
+
+ARCHITECTURES="all !x86_gcc2"
```

Comment:
Shouldn't it be x86_64 !all

Reply:
Actually should be "!all x86_64" then (otherwise !all will take over from x86_64)

Reply:
or remove the `!all` part altogether and just use `"x86_64"`